src/clj/rems/auth/oidc.clj

```diff
@@ -172,11 +196,14 @@
                 user-data (plugins/transform :extension-point/transform-user-data user-data)
                 user (find-or-create-user! user-data)]
 
+            (maybe-check-csrf-token request)
```

Comment:
should we check csrf-token earlier already? before sending request to token endpoint

Reply:
Yeah, I guess that's better.